lithops/standalone/standalone.py

```diff
@@ -211,16 +211,7 @@ def create_runtime(self, runtime):
         Installs the proxy and extracts the runtime metadata and
         preinstalled modules
         """
-        if not self._is_backend_ready():
-            # The VM instance is stopped
-            init_time = time.time()
-            self.backend.start()
-            self._wait_backend_ready()
-            total_start_time = round(time.time()-init_time, 2)
-            logger.info('VM instance ready in {} seconds'.format(total_start_time))
-
-        self._setup_proxy()
-        self._wait_proxy_ready()
```

Comment:
@kpavel For the same reason stated above, `self._setup_proxy` must be always called here